xendit-java-lib/src/main/java/com/xendit/model/DisbursementPH.java

```diff
@@ -0,0 +1,169 @@
+package com.xendit.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.xendit.Xendit;
+import com.xendit.exception.XenditException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+public class DisbursementPH {
+  @SerializedName("id")
+  private String id;
+
+  @SerializedName("reference_id")
+  private String referenceId;
+
+  @SerializedName("currency")
+  private String currency;
+
+  @SerializedName("amount")
+  private Integer amount;
+
+  @SerializedName("channel_code")
+  private String channelCode;
+
+  @SerializedName("description")
+  private String description;
+
+  @SerializedName("status")
+  private String status;
+
+  @SerializedName("created")
+  private Date created;
+
+  @SerializedName("updated")
+  private Date updated;
+
+  // optionals
+  @SerializedName("email_to")
+  private String[] emailTo;
+
+  @SerializedName("email_cc")
+  private String[] emailCC;
+
+  @SerializedName("email_bcc")
+  private String[] emailBcc;
```

Comment:
these would be under `receipt_notification` object right?

```json
receipt_notification: {
    email_to: [],
    email_cc: [],
    email_bcc: []
}
```

Reply:
13b2edf36536b75f0da2e8fe89a3e5aabbd257d9